src/App.css

```diff
@@ -49,20 +49,31 @@ a.nav-title {
   color: rgb(3, 3, 3);
 }
 
+.icon {
+  color: #0290ff;
+  border: solid rgb(197, 196, 196) 1px;
+  border-radius: 300px;
+  padding: 0.5rem;
+}
+
 /* book styles */
 
+.book-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  background: white;
+  margin: 1rem 2rem;
+  padding: 1rem;
+}
+
 .book-item {
```

Comment:
- [ ] Kindly ensure that your design match that of the Zemplin design. Take a look at the width of elements, spaces, font color, and style.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/192103506-3cf9259d-5458-4d37-9d84-7e223c30fb29.PNG)
![Capture1](https://user-images.githubusercontent.com/85095702/192103517-c9ab350c-7228-4ed0-a75e-69128201e2f2.PNG)
Zemplin design
![zemplin](https://user-images.githubusercontent.com/85095702/192103528-1887a80d-2809-4673-9072-6989fda086af.PNG)
![zemplin2](https://user-images.githubusercontent.com/85095702/192103538-a6e3c6ab-be20-48b8-9234-4f250e223c6e.PNG)